hazelcast/src/main/java/com/hazelcast/cache/impl/journal/CacheEventJournalFunctions.java

```diff
@@ -18,8 +18,8 @@
 
 import com.hazelcast.cache.CacheEventType;
 import com.hazelcast.cache.EventJournalCacheEvent;
-import com.hazelcast.internal.journal.DeserializingEntry;
```

Comment:
DeserializingEntry is not used anymore I guess, except the usage in the IDS factory. Can we get rid of it as well? WDYT?

Reply:
Good catch, haven't thought about checking if it can be removed, I suppose it can.

Reply:
b9e70ba7967